src/commands/sourcemaps/upload.rs

```diff
@@ -287,6 +298,7 @@ fn process_sources_from_paths(
         .get_many::<String>("ignore")
         .map(|ignores| ignores.map(|i| format!("!{i}")).collect())
         .unwrap_or_else(Vec::new);
+    let inject_files = !matches.get_flag("no_rewrite");
```

Comment:
```suggestion
    let inject_files = !matches.get_flag("no_inject");
```

I guess?

Reply:
So we should not add a flag, but reuse the existing one?

Reply:
not sure I understand? You did add a new flag, except you were not using it properly?

Reply:
Sorry, I seem to have had the mother of all brainfarts. You're completely right.